### PR TITLE
Add two analysis videos

### PR DESCRIPTION
**Pull request by pwalczysko**
adding the new videos.
cc @will-moore @jburel @sbesson

### CODE REVIEW

**Comment by pwalczysko** [comment]
deployed on http://idr-testing.openmicroscopy.org/ thank you @will-moore

File: idr_gallery/data/tabs.py [hunk 1]
```diff
@@ -27,9 +27,14 @@
...
-                "title": "Compound analysis: SARS-CoV-2",
-                "text": "Build analysis environment. Access metadata and images. Calculate IC50 and download. Explore dose-response curves.",
+                "id": "tOgKlxyyvSc",
+                "title": "Analysis environment setup",
+                "text": "Set-up local or on-the-cloud analysis environment. Access images and metadata using API. Run analysis examples. Add your own code.",
+            },
+            {
+                "id": "z19EE0_2JBs",
+                "title": "Segment image and compare",
+                "text": "Fetch image and labels from IDR. Segment nuclei using StarDist. Compare new labels with original. Save segmentation polygons.",
```
Comment (jburel) [inline idr_gallery/data/tabs.py:37]: * Possible to add link to startDist?
* Save locally segmentation polygons as geojson

File: idr_gallery/data/tabs.py [hunk 2]
```diff
@@ -27,9 +27,14 @@
...
-                "id": "2qeJbudXnp4",
-                "title": "Compound analysis: SARS-CoV-2",
-                "text": "Build analysis environment. Access metadata and images. Calculate IC50 and download. Explore dose-response curves.",
+                "id": "tOgKlxyyvSc",
+                "title": "Analysis environment setup",
+                "text": "Set-up local or on-the-cloud analysis environment. Access images and metadata using API. Run analysis examples. Add your own code.",
+            },
+            {
+                "id": "z19EE0_2JBs",
+                "title": "Segment image and compare",
```
[the remark this answers is not in the document]
Comment (pwalczysko) [reply idr_gallery/data/tabs.py:36]: That would then not match the title inside the video. I would also think that the word "segment" would attract users looking for segmentation opportunities

[the remark this answers is not in the document]
Comment (pwalczysko) [reply idr_gallery/data/tabs.py:36]: Could add a "Validate software package." as the first sentence inside the text ?

[the remark this answers is not in the document]
Comment (jburel) [reply idr_gallery/data/tabs.py:36]: don't redo the video. Adding a sentence will work

**Comment by pwalczysko** [comment]
@will-moore would you please advise about the formatting ? I do not seem to be able to insert a link into the "text" of the video, although I am using the same motives used in the paragraphs above the video, see https://github.com/IDR/idr-gallery/pull/9/commits/58b1c588adeaf947ffceb7977c60b64c00c80c33 which ended up in 
Screenshot 2022-04-29 at 13 00 53

**Comment by pwalczysko** [comment]
I think that https://github.com/IDR/idr-gallery/pull/9/commits/50473d3cb2d19d01b7085410e1f8253e122c28a8 fixes the formatting problem. All is built at http://idr-testing.openmicroscopy.org/.
Relisting for Monday.

**Comment by jburel** [comment]
Thanks @pwalczysko. Looks good.

**Pull request merged**